TeachingPlatform/users/models.py

```diff
@@ -0,0 +1,47 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+from TeachingPlatform.users.managers import CustomUserManager
+
+LESSON_TYPE = (
+    ("B", "Backend"),
+    ("F", "Frontend"),
+    ("D", "Devops"),
+    ("S", "System Administration"),
+    ("C", "Cloud Engineering"),
+)
```

Comment:
This is the old-fasioned way of defining the field choices. Please use `TextChoices` instead: https://docs.djangoproject.com/en/4.1/ref/models/fields/#enumeration-types